cohortextractor/update_vmp_mapping.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import requests
+
+from .mssql_utils import mssql_dbapi_connection_from_url
+
+
+def update_vmp_mapping():
+    """Update VmpMapping table with latest data from OpenCodelists"""
+
+    mapping = [tuple(record) for record in get_vmp_mapping()]
+    temporary_database = os.environ["TEMP_DATABASE_NAME"]
+    conn = mssql_dbapi_connection_from_url(os.environ["DATABASE_URL"])
+    cursor = conn.cursor()
+    cursor.execute(
+        f"""
+IF OBJECT_ID('{temporary_database}..VmpMapping', 'U') IS NOT NULL
+    DROP TABLE {temporary_database}..VmpMapping
+
+CREATE TABLE {temporary_database}..VmpMapping (
+    id VARCHAR(18) COLLATE Latin1_General_CI_AS,
+    prev_id VARCHAR(18) COLLATE Latin1_General_CI_AS,
+)
+        """
+    )
+    cursor.executemany(
+        f"INSERT INTO {temporary_database}..VmpMapping VALUES (%s, %s)",
+        mapping,
+    )
+    conn.commit()
```

Comment:
I assume the intention here is that all this happens in a transaction so the DROP/re-CREATE doesn't disrupt any running queries. Are you confident that's happening here? I remember having to do some dead chicken waving elsewhere to get a proper transaction e.g.
https://github.com/opensafely-core/cohort-extractor/blob/9da8873acbc473f16e536cb241fdadb6e98a0e8e/cohortextractor/tpp_backend.py#L226-L237

You could check by sticking some bad values in `mapping` so that the INSERT blows up and then checking to see if the original table is still there.